python/external_config/external_command.py

```diff
@@ -221,13 +271,55 @@ def pipeline_configuration_name(self):
         """
         return self._pipeline_config_name
 
+    @property
+    def system_name(self):
+        """
+        The system name for the command
+        """
+        return self._callback_name
+
     @property
     def display_name(self):
         """
         Display name, suitable for display in a menu.
         """
         return self._display_name
 
+    @property
+    def group(self):
+        """
+        Group command belongs to or None if not defined.
+        """
+        return self._group
+
+    @property
+    def is_group_default(self):
+        """
+        True if this command is a default action for a group.
+        """
+        return self._is_group_default
+
+    @property
+    def excluded_permission_groups_hint(self):
+        """
+        Legacy option used by some older Shotgun toolkit apps.
+        Apps may hint a list of permission groups for which
+        the app command should not be displayed.
+
+        Returns a list of Shotgun permission groups (as strings)
+        where this command is not appropriate.
+        """
+        return self._sg_deny_permissions or []
+
+    @property
+    def support_shotgun_multiple_selection(self):
+        """
+        Legacy flag indicated by some older Toolkit apps,
```

Comment:
Just checking: we say legacy, does that mean that we don't plan to support this anymore and this is kept for backward compatibility?

Reply:
good point. at some point we considered it legacy, but in recent docs for `register_command` it's not legacy so i'll remove that here.